Hi3Helper.Core/Lang/en.json

```diff
@@ -438,79 +438,79 @@
 
   "_Dialogs": {
     "DeltaPatchDetectedTitle": "Warning: Delta Patch Update is Detected!",
-    "DeltaPatchDetectedSubtitle": "Delta patch will be used to update your game from {0} to {1}.\r\nRemember, you might still need to download some files in-game as usual.\r\n\r\nDo you want to proceed?",
-    "DeltaPatchPrevFailedTitle": "Previous Delta Patch was Failed/Incompleted!",
-    "DeltaPatchPrevFailedSubtitle": "The previous Delta Patch failed to apply. Do you want to restore your game to prevent from redownloading?",
-    "GameConversionPrevFailedTitle": "Previous Game Conversion was Failed/Incompleted!",
+    "DeltaPatchDetectedSubtitle": "Delta patch will be used to update your game from {0} to {1}.\r\nRemember that you may still need to download some files in-game.\r\n\r\nDo you wish to proceed?",
+    "DeltaPatchPrevFailedTitle": "Previous Delta Patch has been incorrectly applied or failed to finish applying!",
+    "DeltaPatchPrevFailedSubtitle": "The previous Delta Patch failed to apply. Do you want to restore your game to prevent a redownload?",
+    "GameConversionPrevFailedTitle": "Previous Game Conversion has Failed or was Incompleted!",
```

Comment:
Maybe making it like `Previous Game Conversion either Failed or Incomplete!` shorten and make it clearer

Reply:
Rephrased it to `Previous Game Conversion either failed or was not completed!` for sentence flow